TemplateExample.swift

```diff
@@ -0,0 +1,48 @@
+// Generated by Unstringify.
+// DO NOT EDIT!
+
+import Foundation
+import Unstringified
+
+private final class _Unstringified {}
+
+extension Unstringified {
+    public var localizableStringsTableName: String? {
+        return nil
+    }
+    
+    public var localizableStringsBundle: Bundle? {
+        let _UnstringifiedBundle = Bundle(for: _Unstringified.self)
+        guard _UnstringifiedBundle.bundleIdentifier != Bundle.main.bundleIdentifier else {
+            return Bundle.main
+        }
+        let bundleURL = _UnstringifiedBundle.bundleURL
+        let bundleName = bundleURL.lastPathComponent
+        let resource = (bundleName as NSString).deletingPathExtension
+        guard let path = _UnstringifiedBundle.path(forResource: resource, ofType: "bundle") else {
+            return nil
+        }
+        return Bundle(path: path)
+    }
+}
+
```

Comment:
In this custom template example I would replace the following lines with:
```
public enum Localizable {
     public enum Text: String, Unstringified {
          public typealias StringType = String
          case $KEYS_ARRAY
      }

       public enum Format: Unstringified {
          public typealias StringType = String
          case $FORMATED_KEYS_ARRAY
      }

       public enum RichText: String, Unstringified {
          public typealias StringType = NSAttributedString
          case $RICH_KEYS_ARRAY
      }

       public enum RichFormat: Unstringified {
          public typealias StringType = NSAttributedString
          case $FORMATED_RICH_KEYS_ARRAY
      }
}
```

Reply:
Done!